lms/djangoapps/instructor/views/api.py

```diff
@@ -2642,3 +2642,21 @@ def mark_student_can_skip_entrance_exam(request, course_id):  # pylint: disable=
         'message': message,
     }
     return JsonResponse(response_payload)
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+@require_POST
+def start_certificate_generation(request, course_id=None):
```

Comment:
`course_id` should not be None in any case. Let me remove its default value if `course_id` param.